aries-site/src/pages/components/pagination.mdx

```diff
@@ -1,42 +1,69 @@
+<Example
+  caption="Diagram illustrating the parts composing Pagination."
+  plain
+>
+  <PaginationAnatomy />
+</Example>
 
-If there is only one page of items, it is optional to display the pagination component and summary of results.
+| Label | Region          | Purpose                                                                            | Required | Notes                                                                                                                                      |
+| :---: | --------------- | ---------------------------------------------------------------------------------- | :------: | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| **1** | **Container**   | Organizes its children, where they appear, how they adapt responsively.            | Required | --                                                                                                                                         |
+| **2** | **Summary**     | Provide clarity on what range of items are in view.                                | Optional | --                                                                                                   |
+| **3** | **StepOptions** | Displays the current number of items within a page and allows the user to modify.  | Optional | Step defaults to 10. Dropdown options default to 10, 25, 50, 100. The default step and step options can be customized to fit the use case. |
+| **4** | **Navigation**  | Allow user to jump to a specific page.                                             | Required | --                                                                               |
 
-### Placement of pagination component
+## Types
 
-The pagination component should be placed underneath and justified to the end of the content it is paginating.
+### Pagination with Standard Navigation
 
-### Showing summary of results
+The standard navigation in the Pagination component displays up to 7 buttons, including Edge Pages (first and last), Middle Pages (maximum of 3 pages) and Arrow Buttons (previous and next).
 
-It may be beneficial to provide users with a summary of what range of results they are viewing. This should take the format of "Showing x-y of z" where x is the start of the page, y is the end of the page, and z is the total number of records. This should be left-aligned in a row with the pagination component.
+### Pagination with Custom Navigation
```

Comment:
Maybe just call this "Custom"? or "Modified middle and edge pages"?

Reply:
I would call it Custom Navigation at least, because we don't have customizations for the other parts of pagination mentioned here - summary and StepOptions